getting-started/data-scientists.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Data scientists
+description: Get started with Coder as a data scientist.
+---
+
+This article will walk you through the process of getting started with a Coder
+workspace capable of supporting data science projects. You'll learn how to:
+
+- Connect Coder to your Git provider;
```

Comment:
maybe here we can have a statement indicating that this guide assumes GitHub, but we also support GitLab and Bitbucket